### ISSUE TITLE
pip install DisplayCAL fails on Windows 11 with Python 3.11.4

### ISSUE THREAD

**Issue by agjell** [eoyilmaz/displaycal-py3#274]
```
C:\Users\dummyuser>pip install DisplayCAL
Collecting DisplayCAL
  Downloading DisplayCAL-3.9.11.tar.gz (10.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.6/10.6 MB 65.6 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [22 lines of output]
      <string>:20: DeprecationWarning: 'msilib' is deprecated and slated for removal in Python 3.13
      Traceback (most recent call last):
        File "C:\Users\dummyuser\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "C:\Users\dummyuser\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\dummyuser\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\dummyuser\AppData\Local\Temp\pip-build-env-srjxy9eb\overlay\Lib\site-packages\setuptools\build_meta.py", line 341, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\dummyuser\AppData\Local\Temp\pip-build-env-srjxy9eb\overlay\Lib\site-packages\setuptools\build_meta.py", line 323, in _get_build_requires
          self.run_setup()
        File "C:\Users\dummyuser\AppData\Local\Temp\pip-build-env-srjxy9eb\overlay\Lib\site-packages\setuptools\build_meta.py", line 338, in run_setup
          exec(code, locals())
        File "<string>", line 1583, in <module>
        File "<string>", line 446, in setup
        File "C:\Users\dummyuser\AppData\Local\Temp\pip-install-g_etflg2\displaycal_6922188531564c3c8a596774680dbf7b\DisplayCAL\util_os.py", line 53, in <module>
          from win32file import *
      ModuleNotFoundError: No module named 'win32file'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by p5k369** [comment]
Hey @agjell note that installing and running displaycal on windows is atm not truly supported. You would have to make changes to several files just in order to make displaycal start on windows and even then you would not be able to perform a calibration.

Independently of that, the error comes from the missing package _pywin32_. This can be solved by `pip install pywin32`

**Comment by agjell** [comment]
Ok, noted :)
Pywin32 was installed, though. So the error must come from elsewhere.

**Comment by q1ngyang** [comment]
I have encountered the same issue as well, ensuring that pywin32 is installed.

`C:\Users\q1ngyang>pip list
Package   Version
--------- -------
pypiwin32 223
pywin32   306

C:\Users\q1ngyang>pip install DisplayCAL
Collecting DisplayCAL
  Using cached DisplayCAL-3.9.11.tar.gz (10.6 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [22 lines of output]
      <string>:20: DeprecationWarning: 'msilib' is deprecated and slated for removal in Python 3.13
      Traceback (most recent call last):
        File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1264.0_x64__qbz5n2kfra8p0\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1264.0_x64__qbz5n2kfra8p0\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1264.0_x64__qbz5n2kfra8p0\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\q1ngyang\AppData\Local\Temp\pip-build-env-37mq2ww7\overlay\Lib\site-packages\setuptools\build_meta.py", line 341, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\q1ngyang\AppData\Local\Temp\pip-build-env-37mq2ww7\overlay\Lib\site-packages\setuptools\build_meta.py", line 323, in _get_build_requires
          self.run_setup()
        File "C:\Users\q1ngyang\AppData\Local\Temp\pip-build-env-37mq2ww7\overlay\Lib\site-packages\setuptools\build_meta.py", line 338, in run_setup
          exec(code, locals())
        File "<string>", line 1583, in <module>
        File "<string>", line 446, in setup
        File "C:\Users\q1ngyang\AppData\Local\Temp\pip-install-jmza9rnf\displaycal_cc6ca13125a5429aa9a5c93cb6ad87a2\DisplayCAL\util_os.py", line 53, in <module>
          from win32file import *
      ModuleNotFoundError: No module named 'win32file'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.`

**Comment by AstralStorm** [comment]
My suspicion is that the setuptools is outdated or broken somehow, and the way exec is done excludes the module.
Maybe use distutils instead?

**Comment by AstralStorm** [comment]
Fixed in my pull request.

**Comment by 515orestis** [comment]
> Fixed in my pull request.

the commit seems to work. clone the project from dev branch 
but when i  try to start DisplayCAL from powershell  this text instead for the appUi

#!C:\Python312\python.exe
# -*- coding: utf-8 -*-

from DisplayCAL.main import main

main()

**Comment by AstralStorm** [comment]
The commit does not fix the executable creation and the shell script is a Bash script, not PowerShell.

You can run DisplayCal in the virtualenv after installing for the moment, using:
```python ./DisplayCal.pyw```

**Comment by eoyilmaz** [comment]
Fixed with #312
